core-plugins/docs/File-batchsink.md

```diff
@@ -28,10 +28,14 @@ If not specified, no suffix is used.
 **jobProperties:** Advanced feature to specify any additional properties that should be used with the sink,
 specified as a JSON object of string to string. These properties are set on the job at runtime. (Macro-enabled)
 
-**format:** The format to write in. Must be 'text', 'avro', or 'parquet'. Defaults to 'text'. (Macro-enabled)
+**Format:** Format to write the records in.
+The format must be one of 'json', 'avro', 'parquet', 'csv', 'tsv', or 'delimited'. (Macro-enabled)
 
-**schema:** The schema to use when writing data in 'avro' or 'parquet' format. Defaults to the input
-schema. (Macro-enabled)
+**Delimiter:** Delimiter to use if the format is 'delimited'.
+The delimiter will be ignored if the format is anything other than 'delimited'.
+
+**Schema:** Schema of the data to write.
+The 'avro' and 'parquet' formats require a schema but other formats do not. (Macro-enabled)
```

Comment:
I think I'll remove this bit about it not being required, since we always want people to set it for lineage purposes (and it is automatically set by the UI)

Reply:
👍